go/vt/vtgate/vindexes/vschema_test.go

```diff
@@ -2033,11 +1765,11 @@ func TestSequence(t *testing.T) {
 	}
 	want := &VSchema{
 		RoutingRules: map[string]*RoutingRule{},
-		uniqueTables: map[string]*Table{
-			"seq":  seq,
-			"t1":   t1,
-			"t2":   t2,
-			"dual": dualb,
+		uniqueTables: map[string]*string{
```

Comment:
@harshit-gangal this is the type of change I don't like having to do. I got tired of fixing tests in the middle of the file, so I kept some of them using golden tests